src/src/WebServer/Metrics.cpp

```diff
@@ -61,130 +61,115 @@
 
 
 void handle_metrics() {
+    TXBuffer.startStream();
+
+
     int result;
     String resultString;
     resultString.reserve(1000);
```

Comment:
This `resultString` can also go as it is no longer needed and with the reserve it still consumes a kB of RAM